src/central-server/admin-service/infra-jpa/src/main/resources/liquibase/centerui/202504241111-security-server-maintenance-mode.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
+                   logicalFilePath="centerui/202504241111-security_server_maintenance_mode.xml">
+
+    <changeSet author="niis" id="202504241111-security_servers-add-new-columns">
+        <comment>Add new columns to security_servers table</comment>
+        <addColumn tableName="security_servers">
```

Comment:
It is probably a good idea to add this change to the data model document as well https://github.com/nordic-institute/X-Road/blob/develop/doc/DataModels/dm-cs_x-road_central_server_configuration_data_model.md#221-security_servers